edgelet/iotedge/src/check/checks/check_fs_calls.rs

```diff
@@ -0,0 +1,62 @@
+use std::fs;
+use std::fs::File;
+use std::io::prelude::*;
+use failure::ResultExt;
+
+use crate::{
+    check::{Check, CheckResult, Checker, CheckerMeta},
+};
+
+#[derive(Default, serde_derive::Serialize)]
+pub(crate) struct CheckFsCalls {}
+
+#[async_trait::async_trait]
+impl Checker for CheckFsCalls {
+    fn meta(&self) -> CheckerMeta {
+        CheckerMeta {
+            id: "check-fs-calls",
+            description: "IoT Edge can perform File System Operations",
+        }
+    }
+
+    async fn execute(&mut self, check: &mut Check) -> CheckResult {
+        self.inner_execute(check)
+            .await
+            .unwrap_or_else(CheckResult::Failed)
+    }
+}
+
+impl CheckFsCalls {
+    #[allow(clippy::unused_self)]
+    #[allow(unused_variables)]
+    async fn inner_execute(&mut self, check: &mut Check) -> Result<CheckResult, failure::Error> {
+        // Test Read file and Remove file
+        File::create("createfile_test.txt").with_context(|_| format!("Unable to create test file for file read test"))?;
```

Comment:
We should try performing all these operations in the home directory. There is no guarantee that the location where the user runs iot edge would be R/W accessible for iotedge user